packages/block-editor/src/components/inserter/media-tab/utils.js

```diff
@@ -5,19 +5,23 @@ import { createBlock } from '@wordpress/blocks';
 
 const mediaTypeTag = { image: 'img', video: 'video', audio: 'audio' };
 
+/** @typedef {import('./hooks').InserterMediaItem} InserterMediaItem */
+
+/**
+ * Creates a block and a preview element from a media object.
+ *
+ * @param {InserterMediaItem} media     The media object to create the block from.
+ * @param {string}            mediaType The media type to create the block for( 'image', 'video', 'audio')
```

Comment:
```suggestion
 * @param {string}            mediaType The media type to create the block for ('image', 'video', 'audio').
```

In another hunk of the diff you used the following narrower type:

```
@property {('image'|'audio'|'video')}                              mediaType
```

Why not use it here? Does JSDoc allow it?

Reply:
Just thought about it in the other interface. I'll update here too.